libraries/standard/mqtt/src/mqtt.c

```diff
@@ -25,6 +25,10 @@
 #include "mqtt.h"
 #include "private/mqtt_internal.h"
 
+
+/* Transport recv retry limit for CONNACK packet. */
+#define CONNACK_RECV_RETRY_LIMIT    3U
```

Comment:
Whenever I read "retry limit" I have to think for a bit, because we count the initial attempt as a "retry". Can the comment reflect that the initial attempt is included in this number?

Reply:
Removed the retry logic.